packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1086,7 +1107,7 @@ oie.enroll.okta_verify.channel.sms.description = Make sure you can access the te
 oie.enroll.okta_verify.channel.sms.description.updated = Make sure you can access the text on the receiving device
 oie.enroll.okta_verify.qrcode.title = Set up Okta Verify via QR code
 oie.enroll.okta_verify.qrcode.step1 = On your mobile device, download the Okta Verify app from the App Store (iPhone and iPad) or Google Play (Android devices).
-oie.enroll.okta_verify.qrcode.step1.updated = On your other device, download the Okta Verify app from the App Store� (iPhone� and iPad�) or on Google Play (Android? devices).
+oie.enroll.okta_verify.qrcode.step1.updated = On your other device, download the Okta Verify app from the App Store� (iPhone� and iPad�) or on Google Play (Android? devices).
```

Comment:
?

Reply:
Hmm, I did not touch this at all, not sure why it changed 🤔 let me try copying from remote and overwriting the update.